src/Microsoft.IdentityModel.Tokens/Validation/Results/Details/ValidationError.cs

```diff
@@ -68,11 +52,11 @@ internal virtual Exception GetException()
             return GetException(ExceptionType, InnerException);
         }
 
-        internal Exception GetException(Type exceptionType, Exception innerException)
+        internal Exception GetException(Type exceptionType, Exception? innerException)
         {
-            Exception exception = null;
+            Exception? exception = null;
 
-            if (innerException == null && InnerValidationError == null)
+            if (innerException is null)
```

Comment:
Why don't we need the InnerValidationError check here?

Reply:
We removed the InnerValidationError from ValidationError since it was only used in one very niche case that we refactored.